megatron/model/transformer.py

```diff
@@ -96,8 +97,9 @@ class ParallelMLP(MegatronModule):
     state back into h hidden dimension.
     """
 
-    def __init__(self, config, moe=False, enable_expert_tensor_parallelism=False):
+    def __init__(self, config : TransformerConfig, moe=False, enable_expert_tensor_parallelism=False):
         super(ParallelMLP, self).__init__()
+        self.config: TransformerConfig = config
```

Comment:
Why is this needed?

Reply:
hi, because the test is using this https://github.com/microsoft/Megatron-DeepSpeed/pull/401/files#diff-4c057a47580dfb962abb8c6940d227718f1646813afcb9d9b24f8bc1f2b6fc64R30. Would you mind enable test with this change or avoid in the case?